examples/timeouts/test.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+const test = require('ava');
+
+const {delay} = require('.');
+
+test('start server', async t => {
+	t.timeout(100);
+
+	await delay(50);
+
+	t.pass();
+});
+
+test('connect with database', async t => {
+	t.timeout(100, 'make sure database has started');
+
+	await delay(200);
+
+	t.pass();
+});
```

Comment:
These kind of tasks you'd typically do in a `before` hook. Which should work with `t.timeout()` but that may be more confusing to people? So perhaps just make something up?

Reply:
Yes you're right. I'll change them.

Reply:
Done. Hope this is a bit better :).